packages/wagmi/src/client.ts

```diff
@@ -32,6 +34,7 @@ import { WALLET_CHOICE_KEY } from './utils/constants.js'
 export interface Web3ModalClientOptions extends Omit<LibraryOptions, 'defaultChain' | 'tokens'> {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   wagmiConfig: Config<any, any>
+  siweConfig?: SIWEControllerClient
```

Comment:
This should probably be of type `Web3ModalSIWEClient` to be more accurate?

Reply:
Not really but I updated the types to be clearer and more flexible.